runtime/src/commonMain/kotlin/pbandk/MessageMap.kt

```diff
@@ -51,6 +51,7 @@ class MessageMap<K, V> internal constructor(override val entries: Set<Entry<K, V
 
             @Suppress("UNCHECKED_CAST")
             override val descriptor: MessageDescriptor<Entry<K, V>> = MessageDescriptor(
+                fullName = "MapFieldEntry",
```

Comment:
Shouldn't this be `google.protobuf.MapFieldEntry`

Reply:
TL;DR: No. This is a fake name because there's no actual protobuf message defined for this class.

This isn't an official type defined by Google. In this case, there's no actual protobuf definition that corresponds to this `MessageMap.Entry` class. According to protobuf semantics, a protobuf `map` field is equivalent on the wire to a `repeated` message field where the message contains a `key` field and `value` field. So in pbandk we fake the protobuf message descriptor that would theoretically exist for the Kotlin `Map.Entry`. But since there's no actual protobuf definition, we have to fake the protobuf message name as well.